src/test/java/com/easypost/Fixture.java

```diff
@@ -73,6 +76,14 @@ public static String reportDate() {
         return "2022-04-12";
     }
 
+    /**
+     * Prefix for report type.
+     * @return report type prefix
+     */
+    public static String reportIdPrefix() {
+        return "shprep_";
```

Comment:
Fixtures are meant to be reusable data that we pass to the API specifically. 

I'd rather we either had this bit in the report test file itself or simply hard-coded the assertion (in this case it should be fine since it's what is always expected and shouldn't change.)